src/directives/profile.js

```diff
@@ -21,6 +21,10 @@ goog.require('ngeo.profile');
  * processed by {@link ngeox.profile.ElevationExtractor} and
  * {@link ngeox.profile.PoiExtractor}.
  *
+ * @htmlAttribute {???} ngeo-profile ???
+ * @htmlAttribute {ngeox.profile.ProfileOptions} ngeo-profile-options The options
+ * @htmlAttribute {???} ngeo-profile-pois ???
+ * @htmlAttribute {???} ngeo-profile-highlight ???
```

Comment:
@htmlAttribute {*} ngeo-profile-highlight Any property on the scope which evaluated value may correspond to distance from origin.